src/components/icon/_scroll-to-top.less

```diff
@@ -0,0 +1,22 @@
+.scroll-to-top-btn {
+  &:before {
+    content: "\f106";
+    font-family: "Font Awesome 5 Free";
+    font-weight: 800;
+    display: inline-block;
+    font-style: normal;
+    font-variant: normal;
+    text-rendering: auto;
+    -webkit-font-smoothing: antialiased;
+  }
+
+  &:not(.rounded-btn) {
+    padding: 0 0.5em;
+  }
+
+  &.rounded-btn {
+    width: 2em;
```

Comment:
change width and height to 1.5em and padding to `0 0 0 1px`

Reply:
Fixed !